crates/volta-core/src/tool/npm/resolve.rs

```diff
@@ -42,11 +42,12 @@ fn fetch_npm_index(hooks: Option<&ToolHooks<Npm>>) -> Fallible<(String, PackageI
     };
 
     let spinner = progress_spinner(&format!("Fetching public registry: {}", url));
-    let metadata: RawPackageMetadata = attohttpc::get(&url)
+    let http_client = Client::new();
+    let metadata: RawPackageMetadata = http_client
+        .get(&url)
         .header(ACCEPT, NPM_ABBREVIATED_ACCEPT_HEADER)
         .send()
-        .and_then(Response::error_for_status)
-        .and_then(Response::json)
+        .and_then(|resp| resp.json())
```

Comment:
Similarly here, we should preserve the `.and_then(Response::error_for_status)` call. I believe we can also keep the existing point-free style of `.and_then(Response::json)` as well, which avoids creating a new closure.